CONTRIBUTING.md

```diff
@@ -70,5 +70,8 @@ The basic steps for working with GitHub Flow is as follows:
   * Push your _feature branch_ to the _origin_.
   * Make a Pull Request for your _feature branch_ to the _upstream_.  
 
+For more information about the Github flow, please refer to Github's [official guide](https://guides.github.com/introduction/flow/) 
+
+Thank you for your interest to contribute and we hope you will find a welcome community that we hoped to create! 
```

Comment:
Thank you for your interest to contribute and we hope you will find a welcoming community that we hoped to create!